src/poetry/console/commands/remove.py

```diff
@@ -83,8 +83,8 @@ def handle(self) -> int:
         if "group" in poetry_content and not poetry_content["group"]:
             del poetry_content["group"]
 
-        removed = set(removed)
-        not_found = set(packages).difference(removed)
+        removed_ = set(removed)
```

Comment:
Not sure I like the underscore suffix here. Why not `removed_set` or something similar?